hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hss/schedulenative/ScheduleNativeTranslator.java

```diff
@@ -0,0 +1,121 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.contract.impl.exec.systemcontracts.hss.schedulenative;
+
+import static com.hedera.node.app.service.contract.impl.exec.systemcontracts.HtsSystemContract.HTS_EVM_ADDRESS;
+import static com.hedera.node.app.service.contract.impl.exec.systemcontracts.hts.create.CreateTranslator.createSelectorsMap;
+import static com.hedera.node.app.service.contract.impl.exec.systemcontracts.hts.update.UpdateTranslator.updateSelectorsMap;
+import static com.hedera.node.app.spi.workflows.HandleException.validateTrue;
+import static java.util.Objects.requireNonNull;
+import static org.hyperledger.besu.datatypes.Address.fromHexString;
+
+import com.esaulpaugh.headlong.abi.Address;
+import com.esaulpaugh.headlong.abi.Function;
+import com.hedera.hapi.node.base.AccountID;
+import com.hedera.hapi.node.base.ResponseCodeEnum;
+import com.hedera.hapi.node.transaction.TransactionBody;
+import com.hedera.node.app.service.contract.impl.exec.gas.DispatchType;
+import com.hedera.node.app.service.contract.impl.exec.gas.SystemContractGasCalculator;
+import com.hedera.node.app.service.contract.impl.exec.systemcontracts.common.AbstractCallTranslator;
+import com.hedera.node.app.service.contract.impl.exec.systemcontracts.common.Call;
+import com.hedera.node.app.service.contract.impl.exec.systemcontracts.hss.HssCallAttempt;
+import com.hedera.node.app.service.contract.impl.exec.systemcontracts.hts.HtsCallFactory;
+import com.hedera.node.app.service.contract.impl.exec.systemcontracts.hts.ReturnTypes;
+import com.hedera.node.app.service.contract.impl.hevm.HederaWorldUpdater;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Arrays;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.apache.tuweni.bytes.Bytes;
+
+@Singleton
+public class ScheduleNativeTranslator extends AbstractCallTranslator<HssCallAttempt> {
+
+    public static final Function SCHEDULED_NATIVE_CALL =
+            new Function("scheduleNative(address,bytes,address)", ReturnTypes.RESPONSE_CODE_ADDRESS);
+    private static final int SCHEDULE_CONTRACT_ADDRESS = 0;
+    private static final int SCHEDULE_CALL_DATA = 1;
+    private static final int SCHEDULE_PAYER = 2;
+
+    private final HtsCallFactory htsCallFactory;
+
+    @Inject
+    public ScheduleNativeTranslator(@NonNull final HtsCallFactory htsCallFactory) {
+        requireNonNull(htsCallFactory);
+        this.htsCallFactory = htsCallFactory;
+    }
+
+    @Override
+    public boolean matches(@NonNull final HssCallAttempt attempt) {
+        return attempt.isSelector(SCHEDULED_NATIVE_CALL) && innerCallValidation(attempt);
```

Comment:
Do we want a feature flag to guard this here?

Reply:
We can reuse the `contracts.systemContract.scheduleService.enabled` ?

Reply:
created a new config.  we want to be able to just turn this off instead of all schedule system contract functionality.